katalog/fluentd/fluent.conf

```diff
@@ -1,69 +1,42 @@
-@include systemd.conf
+@include "#{ENV['FLUENTD_SYSTEMD_CONF'] || 'systemd'}.conf"
+@include "#{ENV['FLUENTD_PROMETHEUS_CONF'] || 'prometheus'}.conf"
 @include kubernetes.conf
-
-<system>
-  log_level error
-</system>
-
-<filter kubernetes.var.log.containers.nginx-ingress-**.log>
-  @type parser
-  format /^(?<host>[^ ]*) (?<domain>[^ ]*) \[(?<x_forwarded_for>[^\]]*)\] (?<server_port>[^ ]*) (?<user>[^ ]*) \[(?<time>[^\]]*)\] "(?<method>\S+[^\"])(?: +(?<path>[^\"]*?)(?: +\S*)?)?" (?<code>[^ ]*) (?<size>[^ ]*)(?: "(?<referer>[^\"]*)" "(?<agent>[^\"]*)")? (?<request_length>[^ ]*) (?<request_time>[^ ]*) (?:\[(?<proxy_upstream_name>[^\]]*)\] )?(?<upstream_addr>([^ ,]+(,\s)?)+) (?<upstream_response_length>([^ ,]+(,\s)?)+) (?<upstream_response_time>([^ ,]+(,\s)?)+) (?<upstream_status>([^ ,]+(,\s)?)+) (?<request_id>[^ ]*)$/
-  time_format %d/%b/%Y:%H:%M:%S %z
-  key_name log
-  types server_port:integer,code:integer,size:integer,request_length:integer,request_time:float,upstream_response_length:integer,upstream_response_time:float,upstream_status:integer
-  reserve_data yes
-</filter>
+@include conf.d/*.conf
 
 <match **>
    @type elasticsearch
    @id out_es
-   log_level info
+   @log_level info
    include_tag_key true
    host "#{ENV['FLUENT_ELASTICSEARCH_HOST']}"
    port "#{ENV['FLUENT_ELASTICSEARCH_PORT']}"
+   path "#{ENV['FLUENT_ELASTICSEARCH_PATH']}"
    scheme "#{ENV['FLUENT_ELASTICSEARCH_SCHEME'] || 'http'}"
    ssl_verify "#{ENV['FLUENT_ELASTICSEARCH_SSL_VERIFY'] || 'true'}"
-   user "#{ENV['FLUENT_ELASTICSEARCH_USER']}"
-   password "#{ENV['FLUENT_ELASTICSEARCH_PASSWORD']}"
-   reload_connections "#{ENV['FLUENT_ELASTICSEARCH_RELOAD_CONNECTIONS'] || 'true'}"
+   ssl_version "#{ENV['FLUENT_ELASTICSEARCH_SSL_VERSION'] || 'TLSv1'}"
```

Comment:
Please use TLSv1_2 as is the default for the es plugin [here](https://github.com/uken/fluent-plugin-elasticsearch#clienthost-certificate-options)